src/common/i18n/translations/en.json

```diff
@@ -0,0 +1,101 @@
+{
+  "loginScanCard": {
+    "loginWithQR": "Please log in with your unique QR code provided by your supervisor",
+    "scanToLogin": "Scan to login",
+    "needHelp": "Need help?",
+    "askQuestion": "Ask a question"
+  },
+  "loginMobileNumberCard": {
+    "enterMobileNumber": "Please enter your mobile phone number to receive a one-time password.",
+    "mobileNumber": "Mobile phone number",
+    "sendOtp": "Send OTP"
+  },
+  "loginOTPCard": {
+    "sendingOtp": "We're sending you the one-time password...",
+    "otp": "OTP",
+    "resendIn": "Resend in %{ss}s",
+    "submit": "Submit",
+    "resend": "Resend"
+  },
+  "collectCustomerDetailsScreen": {
+    "checkEligibleItems": "Check the number of item(s) eligible for redemption",
+    "scanIdentification": "Scan identification",
+    "or": "Or",
+    "enterIdNumber": "Enter ID Number",
+    "check": "Check"
+  },
+  "idScanner": {
+    "enterIdManually": "Enter ID manually",
+    "enterManually": "Enter manually",
+    "enterVoucherId": "Enter voucher ID",
+    "scanToCheck": "Scan to check validity"
+  },
+  "navigationDrawer": {
+    "logout": "Logout",
+    "addCampaign": "Add Campaign",
+    "selectCampaign": "Select campaign",
+    "changeChampaign": "Change Campaign",
+    "helpSupport": "Help & Support",
+    "termsOfUse": "Terms of use",
+    "privacyStatement": "Privacy Statement",
+    "reportVulnerability": "Report Vulnerability"
+  },
+  "customerQuotaScreen": {
+    "campaignExpiredOn": "Expired on %{dateTime}",
+    "campaignValidTo": "Valid till %{dateTime}",
+    "quotaRedeemedLimitReachedIDNumber": "Identification number(s)",
+    "quotaLimitMax": "Max",
+    "quotaAppealCancel": "Cancel",
+    "quotaButtonCheckout": "Checkout",
+    "quotaButtonAdd": "+ Add",
+    "quotaButtonAddVoucher": "Add voucher",
+    "quotaAddId": "Add another ID to combine customer quotas",
+    "quotaCategoryVouchers": "Voucher(s):",
+    "quotaIdentifierButtonScan": "Scan",
```

Comment:
I think this is a dynamic copy. See `policies[x].identifiers[y].scanButton.text`

Reply:
hmm i think it's for a fallback 😮
`
 text={
   scanButton.text ||
    i18n.t("customerQuotaScreen.quotaIdentifierButtonScan")
  }`

Reply:
I see!!